lua/wire/client/text_editor/texteditor.lua

```diff
@@ -2506,6 +2506,14 @@ function EDITOR:AC_Check( notimer )
 	self:AC_SetVisible( false )
 end
 
+local function concat_args(args_table, where, cend)
+	local res = ""
+	for k, v in ipairs(args_table) do
+		res = res .. v[where] .. cend
```

Comment:
String concatenation is incredibly slow in lua to build a buffer because strings are immutable (See my above review, just use table.concat as before.)